crates/store/re_video/src/decode/ffmpeg.rs

```diff
@@ -546,7 +560,32 @@ fn read_ffmpeg_output(
             }
 
             FfmpegEvent::ParsedVersion(ffmpeg_version) => {
-                re_log::debug_once!("FFmpeg version is: {}", ffmpeg_version.version);
+                re_log::debug_once!("FFmpeg version is {}", ffmpeg_version.version);
+
+                let mut version_parts = ffmpeg_version.version.split(".");
+                if let (Some(major), Some(minor)) = (
+                    version_parts
+                        .next()
+                        .and_then(|part| part.parse::<u32>().ok()),
+                    version_parts
+                        .next()
+                        .and_then(|part| part.parse::<u32>().ok()),
+                ) {
+                    if major < FFMPEG_MINIMUM_VERSION_MAJOR
+                        || (major == FFMPEG_MINIMUM_VERSION_MAJOR
+                            && minor < FFMPEG_MINIMUM_VERSION_MINOR)
+                    {
+                        re_log::warn_once!(
+                            "FFmpeg version is {}. Only versions >= {FFMPEG_MINIMUM_VERSION_MAJOR}.{FFMPEG_MINIMUM_VERSION_MINOR} are officially supported",
```

Comment:
```suggestion
                            "The found `ffmpeg` binary on your system has version {}. Rerun has only been tested with >= {FFMPEG_MINIMUM_VERSION_MAJOR}.{FFMPEG_MINIMUM_VERSION_MINOR}. Please update your `ffmpeg` version.",
```

Reply:
Even better: include the link to the latest ffmpeg version, as provided by `ffmpeg-sidecar`!

Reply:
Don't want to return an error since chances that things just work are quite high. But yeah let's log an error with url to make it more visible!

Reply:
changed my mind again after seeing that `7.1-essentials_build-www.gyan.dev` is a version string we might get. This is too wild to make it any of this an error, stays a warning
But adding the download link